Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -320,6 +322,7 @@ class ComputedValues {
         float font_size { InitialValues::font_size() };
         int font_weight { InitialValues::font_weight() };
         CSS::FontVariant font_variant { InitialValues::font_variant() };
+        Optional<CSS::CaptionSide> caption_side { InitialValues::caption_side() };
```

Comment:
Why is this `Optional`? There is always a computed value for `caption-side` no?

Reply:
You're right, got confused and followed `value_id_to_caption_side` mechanically. Fixed.